arch/arm64/rules.mk

```diff
@@ -91,6 +91,7 @@ TOOLCHAIN_PREFIX := $(ARCH_$(ARCH)_TOOLCHAIN_PREFIX)
 $(info TOOLCHAIN_PREFIX = $(TOOLCHAIN_PREFIX))
 
 ARCH_COMPILEFLAGS += $(ARCH_$(ARCH)_COMPILEFLAGS)
+ARCH_COMPILEFLAGS += -fno-omit-frame-pointer -fno-optimize-sibling-calls
```

Comment:
Not so sure about these, was this accidentally included? Omit frame pointer I can see, but the optimize sibling calls may be kind of a performance hit?

Reply:
I had referred to Linux implementation for these flags. True, there is a performance hit though this could allow pinpointing the recursion depth properly with this flag. Will remove this one since its not too critical to have this option passed to the compiler.